tests/collectioncardmakertest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+require('../src/CollectionCardMaker.php');
+
+use PHPUnit\Framework\TestCase;
+
+class CollectionCardMakerTest extends TestCase {
+
+
+public function testcreateGameCard()
+{
```

Comment:
method should be indented further (and all code that follows) because it's inside a class

Reply:
also because it's a unit test, the method should be named like this: test_success_create_game_card

so that you can run it with this snazzy command:

```
phpunit CollectionCardMakerTest.php --testdox
```